components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/utils/APIUtil.java

```diff
@@ -2505,63 +2505,65 @@ public static boolean isAnalyticsEnabled() {
         return propertyMap;
     }
 
+	public static void addDefinedAllSequencesToRegistry(UserRegistry registry,
+	                                                    String customSequenceType)
+			throws APIManagementException {
+
+		try {
+			File inSequenceDir = new File(
+					APIConstants.API_CUSTOM_SEQUENCES_FOLDER_LOCATION + File.separator +
+					customSequenceType);
+			File[] sequences = inSequenceDir.listFiles();
+
+			for (File sequenceFile : sequences) {
+				String sequenceFileName = sequenceFile.getName();
+				String regResourcePath =
+						APIConstants.API_CUSTOM_SEQUENCE_LOCATION + File.separator +
+						customSequenceType + File.separator + sequenceFileName;
+				if (registry.resourceExists(regResourcePath)) {
+					if (log.isDebugEnabled()) {
+						log.debug("Defined sequences have already been added to the registry");
+					}
+				} else {
+					if (log.isDebugEnabled()) {
+						log.debug("Adding defined sequences to the registry.");
+					}
+
+					InputStream inSeqStream =
```

Comment:
it is better if the debug message is changed to something as, "The sequence file with the name + sequenceFileName + already exists in the registry" in oppose to mentioning the defined sequences already reside in the registry.

Also when adding it is better to mention that a file with the  specific file name is being added to the registry, instead of mentioning that the sequences are added to the registry.